### PR TITLE
D2k extended upgradetooltips

### PR DESCRIPTION
**Pull request by ltem**
PR for #12069 

Quite simple changes to the d2k yaml-files.

I added also a small fix for the combat tank tooltip:

Before            |  Now          
:-------------------------:|:-------------------------:
openra-2017-01-05t221338150z  |  openra-2017-01-05t214147078z

If unwanted it will be deleted.

### CODE REVIEW

File: mods/d2k/rules/structures.yaml [hunk 1]
```diff
@@ -23,6 +23,8 @@ concretea:
 	Building:
 		Footprint: xx xx
 		Dimensions: 2,2
+	Tooltip:
+		Name: Concret Slab
```
Comment (pchote) [inline mods/d2k/rules/structures.yaml]: Missing an 'e' here.

Comment (ltem) [inline mods/d2k/rules/structures.yaml]: I fixed it by changing the file and  using `git rebase -i HEAD~3` `pick` -> `fixup` command. 
Is there a better way?

**Comment by ltem** [comment]
Don't merge - Additional changes possible 
(IX Research Centre -> IX Research Center to have it consistently in US English)

**Comment by ltem** [comment]
uaaah sorry I will try to fix this :S

**Review by reaperrr — APPROVED** [review]
👍

File: mods/d2k/rules/misc.yaml [hunk 2]
```diff
@@ -259,7 +259,7 @@ upgrade.heavy:
 		BuildLimit: 1
 		BuildDuration: 380
 		BuildDurationModifier: 40
-		Description: Unlocks advanced technology and heavy weapons
+		Description: Unlocks additional construction options    \n(Repair Pad, IX Research Centre) \n\nUnlocks additional heavy units \n(Siege Tank, Missile Tank, MCV)
```
Comment (GraionDilach) [inline mods/d2k/rules/misc.yaml]: Still Centre.

**Review by GraionDilach — APPROVED** [review]
:+1: after the nit addressed.

**Comment by ltem** [comment]
nit fixed

**Comment by reaperrr** [comment]
Thanks!

[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/0a36f868a55161e6b4d87235f2ec63e3ca1b57e0...31cc191f1662ca805132817b38f060e0588ddc0a)

**Pull request merged**